src/components/WwwFrame/TheFooter.vue

```diff
@@ -1,32 +1,36 @@
 <template>
-	<footer class="www-footer" :style="cssVars">
+	<footer class="tw-min-h-[17.5rem] lg:tw-min-h-[25rem] tw-p-2 md:tw-p-3
+				lg:tw-px-2 lg:tw-py-3 tw-bg-tertiary tw-text-primary print:tw-p-0 print:tw-font-book print:tw-h-auto">
```

Comment:
Should `print:tw-min-h-[auto]` be used instead of `print:tw-h-auto`? I think the height will be auto by default and it's just the min-height that needs to be reset for print.

Reply:
Done!